azimuth/routers/v1/utterances.py

```diff
@@ -264,6 +265,47 @@ def get_utterances(
     )
 
 
+@router.post(
+    "",
+    summary="Patch utterances",
+    description="Patch utterances, such as updating proposed actions.",
+    tags=TAGS,
+    response_model=List[UtterancePatch],
+)
+def patch_utterances(
+    request_data: List[UtterancePatch] = Body(...),
+    dataset_split_manager: DatasetSplitManager = Depends(get_dataset_split_manager),
+    task_manager: TaskManager = Depends(get_task_manager),
+) -> List[UtterancePatch]:
+    persistent_ids = [utt.persistent_id for utt in request_data]
+    try:
+        row_indices = dataset_split_manager.get_row_indices_from_persistent_id(persistent_ids)
+    except ValueError as e:
+        raise HTTPException(HTTP_404_NOT_FOUND, detail=f"Persistent id not found: {e}.")
+
+    request_data_complete = {}
```

Comment:
Ah, I see what you mean by `_complete`, but it was more confusing to me than helpful when I first read the name. Like is it adding the "complete" list of utterances? Or adding all utterances' fields? I think I would have caught it if it was named `tags`, in the sense that we are converting to the format of our `tags`. What do you think?
```suggestion
    tags = {}
```

Reply:
I like it, thanks!

Reply:
Actually, I think I prefer `data_actions`